src/docs/product/stats/index.mdx

```diff
@@ -12,7 +12,7 @@ The "Usage Stats" tab shows your organization’s event usage, while also provid
 
 ## Usage Stats
 
-The "Usage Stats" tab shows you the events Sentry has received across your entire organization along with a list of all projects and their consumption of errors, transactions, and attachments. Events fall into one of three categories: _Accepted_, _Filtered_, or _Dropped_. Filtered events are events that were blocked due to your [inbound data filter](/product/data-management-settings/filtering/) rules. Dropped events are ones the system dropped or discarded due to rate limits, quotas, spike protection, or because of invalid data.
+The Usage Stats tab shows you the events Sentry has received across your entire organization along with a list of all projects and their consumption of errors, transactions, and attachments. Events fall into one of three categories: _Accepted_, _Filtered_, or _Dropped_. Filtered events are events that were blocked due to your [inbound data filter](/product/data-management-settings/filtering/) rules. Dropped events are ones the system dropped or discarded due to rate limits, quotas, spike protection, or because of invalid data.
```

Comment:
This is a style thing to add consistency to the docs. UI elements below the page level get quotation marks.

```suggestion
The "Usage Stats" tab shows you the events Sentry has received across your entire organization along with a list of all projects and their consumption of errors, transactions, and attachments. Events fall into one of three categories: _Accepted_, _Filtered_, or _Dropped_. Filtered events are events that were blocked due to your [inbound data filter](/product/data-management-settings/filtering/) rules. Dropped events are ones the system dropped or discarded due to rate limits, quotas, spike protection, or because of invalid data.
```